lib/std/private/strimpl.nim

```diff
@@ -0,0 +1,41 @@
+func toLowerAscii*(c: char): char {.inline.} =
+  if c in {'A'..'Z'}:
+    result = chr(ord(c) + (ord('a') - ord('A')))
+  else:
+    result = c
+
+template cmpIgnoreStyleImpl*(a, b: typed, caseSensitive: static bool = false) =
+  # a, b are string or cstring
+  when caseSensitive:
+    if a[0] != b[0]: return 1
+  var i = 0
+  var j = 0
+  while true:
+    while i < a.len and a[i] == '_': inc i
+    while j < b.len and b[j] == '_': inc j
```

Comment:
Would be nice to have an `static bool` to not have those 2 lines, what if I know that theres no `'_'` used ever?.

Reply:
Feel free to change that in the future PR because modules in private/ could be changed at your will :)